include/itkCoocurrenceTextureFeaturesImageFilter.h

```diff
@@ -82,7 +82,8 @@ namespace Statistics
  **/
 
 template< typename TInputImage,
-          typename TOutputImage>
+          typename TOutputImage,
+          typename TMaskImage = TInputImage>
```

Comment:
Would it make sense to use a default type like "itk::Image<unsigned char,TInputImage::Dimension>" or something similar instead of TInputImage? I quickly seached in ITK and didn't see any filter that used any of these two paradigms for the mask type.

Reply:
I actually found examples of both in ITK:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Segmentation/ConnectedComponents/include/itkScalarConnectedComponentImageFilter.h#L109

https://itk.org/Doxygen/html/itkDiffusionTensor3DReconstructionImageFilter_8h_source.html